src/odemis/acq/align/meteor_super_z.py

```diff
@@ -0,0 +1,155 @@
+import logging
```

Comment:
That file name "meteor_super_z" is a little too specific. It should be working also on the ENZEL and MIMAS. In addition, in the rest of the GUI, we've used less commercial terms, like "Z localization". I'd suggest to rename it "z_localization".